### PR TITLE
Windows: Set window icon

### PR DESCRIPTION
**Pull request by adrido**
This sets the window icon on Windows, that is visible int ALT-TAB Task-switcher, window Title-bar and Task-Manager.

### CODE REVIEW

**Comment by nerzhul** [comment]
seems you broke windows build but not linux :)

**Comment by adrido** [comment]
Forgott that gcc doesent like enum types 😁 . It should work now.

**Comment by Fixer-007** [comment]
Will check this.

**Comment by Fixer-007** [comment]
Works for me! Now I have proper icon everywhere, win7sp1, linux mingw64 crosscompilation to windows.

**Comment by SmallJoker** [comment]
Works fine here, the title bar image is corrected now. The task bar icon was already correct before. (MSVC 2010)
prev

File: src/porting.cpp [hunk 1]
```diff
@@ -627,6 +627,36 @@ bool setXorgWindowIcon(IrrlichtDevice *device)
...
+	case video::EDT_DIRECT3D8:
+		hWnd = reinterpret_cast<HWND>(exposedData.D3D8.HWnd);
+		break;
+	case video::EDT_DIRECT3D9:
+		hWnd = reinterpret_cast<HWND>(exposedData.D3D9.HWnd);
+		break;
+	case video::EDT_OPENGL:
+		hWnd = reinterpret_cast<HWND>(exposedData.OpenGLWin32.HWnd);
+		break;
+	default:
```
Comment (SmallJoker) [inline src/porting.cpp]: Do the software renders not support icons?

Comment (adrido) [inline src/porting.cpp]: No. Thats not available. http://irrlicht.sourceforge.net/docu/structirr_1_1video_1_1_s_exposed_video_data.html

Comment (SmallJoker) [inline src/porting.cpp]: Alright then :+1:

**Comment by adrido** [comment]
> The task bar icon was already correct before. (MSVC 2010)

The taskbar is showing the Application Icon. That did already work. But the (Windows 10) Task-**Manager** 😉  shows the correct icon now too

**Pull request merged**